Tasks/XamariniOS/task.json

```diff
@@ -97,12 +97,24 @@
             "groupName": "advanced"
         },
         {
+            "name": "buildTool",
+            "type": "radio",
+            "label": "Build Tool",
+            "required": false,
+            "groupName": "advanced",
+            "defaultValue": "xbuild",
+            "options": {
+                "xbuild": "xbuild (Xamarin Studio)",
+                "msbuild": "msbuild (Visual Studio for Mac)"
```

Comment:
Even though the executable is "msbuild", I think we should use the official product form of the name here: "MSBuild"